tensorflow/python/platform/BUILD

```diff
@@ -33,6 +33,14 @@ py_library(
     srcs_version = "PY3",
 )
 
+cc_binary(
```

Comment:
Sorry for the delay! This failed an internal check. We prohibit usage of `cc_binary` here because the rule could fail with undefined symbols when TensorFlow is linked to other repositories downstream. 

I can think of two fixes
1. Change to `tf_cc_shared_object` instead (or `tf_cc_binary` if we don't actually need `linkshared`). 
2. Use `tf_native_cc_binary` since this target (and `//tensorflow/core/platform:cpu_feature_guard`) doesn't really depend on TensorFlow targets. 

I've tried Fix#1 and IIRC 3 test targets got a segfault:
```
bazel test --config=opt //tensorflow/python/distribute:packed_distributed_variable_test
bazel test --config=opt //tensorflow/python/distribute:values_v2_test 
bazel test --config=opt //tensorflow/python/eager:function_test
```

Fix#2 caused some internal tests to fail. I'm still investigating why. Would you mind helping take a look at #1 while I look at #2? Thank you very much!